bin/cylc-help

```diff
@@ -332,7 +331,13 @@ comsum = {}
 comsum['test-battery'] = 'Run a battery of self-diagnosing test suites'
 comsum['import-examples'] = 'Import example suites your suite run directory'
 comsum['check-software'] = 'Check required software is installed'
+<<<<<<< HEAD
```

Comment:
Rebase error!

Reply:
Ha! You got me! hahaha

Reply:
Been there myself!

Reply:
:grin:

Reply:
Love git. Changed the file and rebased fixing-up the commit. It should simply disappear like magic... once I push force :grin:

Reply:
Yeah, brilliant,